ui/CONTRIBUTING.md

```diff
@@ -16,13 +16,13 @@ configuration, development, usage, extension and testing.
 It's very important that we can easily track bug fix commits, so their hashes should remain the same in all branches.
 Therefore, a pull request (PR) that fixes a bug, should be sent against a release branch.
 This can be either the "current release" or the "previous release", depending on which ones are maintained.
-Since the goal is a stable master, bug fixes should be "merged forward" to the next branch in order: "previous release" -> "current release" -> master (in other words: old to new)
```

Comment:
the ui/CONTRIBUTING.md can be removed as this was from old primate repo which is not necessary; the repo already has a CONTRIBUTING.md in root repo dir.

Reply:
> the ui/CONTRIBUTING.md can be removed as this was from old primate repo which is not necessary; the repo already has a CONTRIBUTING.md in root repo dir.

@rhtyd Revert the changes, and created a PR (https://github.com/apache/cloudstack/pull/5054) to remove the file 'ui/CONTRIBUTING.md'.